server/graphql/general/src/queries/login.rs

```diff
@@ -88,6 +88,12 @@ pub async fn login(ctx: &Context<'_>, username: &str, password: &str) -> Result<
                 LoginError::DatabaseError(_) => {
                     StandardGraphqlError::InternalError(formatted_error)
                 }
+                LoginError::FetchUserError(_) => {
```

Comment:
I guess i could have kept InternalError when refactoring some of service/login.rs, but this seemed more structured (checked, it doesn't look like any sensitive info is being leaked)